src/high-level/scripts/papi_hl_output_writer.py

```diff
@@ -7,21 +13,31 @@
 import json
 # Make it work for Python 2+3 and with Unicode
 import io
+##\cond
```

Comment:
##\cond and ##\endcond, are used such that Doxygen will ignore a code block. This was done in two places within `papi_hl_output_writer.py`, due to unwanted variables showing up in the variables documentation.